packages/components/src/duotone-picker/color-list-picker/style.scss

```diff
@@ -7,12 +7,6 @@
 	margin: $grid-unit-10 0;
 }
 
-.components-color-list-picker__swatch-button {
-	// Used to simulate styles as a .button.has-icon (which this component can't
-	// opt into, because it has to show more than a simple SVG as the "icon")
```

Comment:
This is untrue, because the `icon` prop renders into an `Icon` component internally, which does support arbitrary components.